test/vesting/VestingWallet.recovery.test.js

```diff
@@ -0,0 +1,77 @@
+const hre = require('hardhat');
+const { ethers } = hre;
+const { expect } = require('chai');
+const { prepare, deployUpgradeable, performUpgrade } = require('../fixture');
+const utils = require('../../scripts/utils');
+
+const allocation = {
+    start: utils.dateToTimestamp('2021-09-01T00:00:00Z'),
+    cliff: utils.durationToSeconds('1 year'),
+    duration: utils.durationToSeconds('4 years'),
+};
+
+describe('VestingWallet ', function () {
+    prepare();
+
+    describe('Vesting recovery', function () {
+        describe('vesting with admin', function () {
+            beforeEach(async function () {
+                allocation.beneficiary = this.accounts.user1.address;
+                allocation.newBeneficiary = this.accounts.user2.address;
+                allocation.owner = this.accounts.admin.address;
+
+                this.vesting = await deployUpgradeable(
+                    hre,
+                    'VestingWallet',
```

Comment:
This seems to be deploying "V0" of the vesting wallet: https://github.com/forta-network/forta-contracts/blob/7d5489a776d4741441296f8e091c7314966e82d2/contracts/vesting/_old/vesting_wallet/VestingWalletV0.sol#L12

I think it should be `VestingWalletV2` ?

Reply:
The wallets we are targeting are using V0.

Not that the storage layout described in `VestingWalletRecoveryLight` is common to V0, V1 and V2.